src/extension.ts

```diff
@@ -77,54 +76,49 @@ export async function activate(context: vscode.ExtensionContext) {
         registerCommand({
             command: 'aws.login',
             callback: async () => await ext.awsContextCommands.onCommandLogin(),
-            telemetryName: {
-                namespace: TelemetryNamespace.Aws,
-                name: 'credentialslogin'
-            }
+            telemetryName: 'aws_credentialslogin'
         })
 
         registerCommand({
             command: 'aws.credential.profile.create',
             callback: async () => await ext.awsContextCommands.onCommandCreateCredentialsProfile(),
-            telemetryName: {
-                namespace: TelemetryNamespace.Aws,
-                name: 'credentialscreate'
-            }
+            telemetryName: 'aws_credentialscreate'
         })
 
         registerCommand({
             command: 'aws.logout',
             callback: async () => await ext.awsContextCommands.onCommandLogout(),
-            telemetryName: {
-                namespace: TelemetryNamespace.Aws,
-                name: 'credentialslogout'
-            }
+            telemetryName: 'aws_credentialslogout'
         })
 
         // register URLs in extension menu
         registerCommand({
             command: 'aws.help',
             callback: async () => {
                 vscode.env.openExternal(vscode.Uri.parse(documentationUrl))
-            }
+            },
+            telemetryName: 'Command_aws.help'
         })
         registerCommand({
             command: 'aws.github',
             callback: async () => {
                 vscode.env.openExternal(vscode.Uri.parse(githubUrl))
-            }
+            },
+            telemetryName: 'Command_aws.github'
         })
         registerCommand({
             command: 'aws.reportIssue',
             callback: async () => {
                 vscode.env.openExternal(vscode.Uri.parse(reportIssueUrl))
-            }
+            },
+            telemetryName: 'Command_aws.reportIssue'
         })
         registerCommand({
             command: 'aws.quickStart',
             callback: async () => {
                 await showQuickStartWebview(context)
-            }
+            },
+            telemetryName: 'Command_aws.quickStart'
```

Comment:
Potentially ditto above? Worst case scenario, this will be a VS Code-specific construct and we can remove `aws`